web/src/pages/FaceLibrary.tsx

```diff
@@ -422,16 +471,20 @@ function TrainingGrid({
           </DialogHeader>
           <div className="flex flex-col gap-1.5">
             <div className="text-sm text-primary/40">{t("details.person")}</div>
-            <div className="text-sm capitalize">{selectedEvent?.name}</div>
-          </div>
-          <div className="flex flex-col gap-1.5">
-            <div className="text-sm text-primary/40">
-              {t("details.confidence")}
-            </div>
             <div className="text-sm capitalize">
-              {(selectedEvent?.score || 0) * 100}%
+              {selectedEvent?.sub_label ?? "Unknown"}
             </div>
           </div>
+          {selectedEvent?.data.sub_label_score && (
+            <div className="flex flex-col gap-1.5">
+              <div className="text-sm text-primary/40">
+                {t("details.confidence")}
+              </div>
+              <div className="text-sm capitalize">
+                {(selectedEvent?.data?.sub_label_score || 0) * 100}%
```

Comment:
This could do with a `Math.round()` as I'm occasionally getting the following

![image](https://github.com/user-attachments/assets/7e7c98c9-f14f-4104-abac-7dac4c563c40)